index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio: setup and mobile version skeleton</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+    <main class="container">        
+        <header>
+            <div class="toolbar">
+                <h2>My Logo</h2>            
+                <img src="images/union.svg" alt="menu" />                 
```

Comment:
- The logo should have a tag link. This is going to become your logo, so plan it in such a way.
- The hamburger menu should be inside a nav tag